src/api/common/schemas/ledger-options.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "ledger-options",
+  "description": "Options for getLedger",
+  "type": "object",
+  "properties": {
+    "ledgerVersion": {"$ref": "ledgerVersion"},
+    "includeAllData": {"type": "boolean"},
+    "includeTransactions": {"type": "boolean"},
+    "includeAccounts": {"type": "boolean"}
```

Comment:
Although, `ledger` is implied by the context, so maybe I like that more cause it's short. Shrug.